_layouts/post.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+{% include head.html %}
+
+<body id="page-top">
+
+    {% include nav_light_bg.html %}
+
+    <section id="blog-post" class="blog-post container" style="padding-right:150px; padding-left:150px">
+        <h1>{{ page.title }}</h1>
+        <small>Published on {{ page.date | date: "%B %d, %Y" }}</small>
```

Comment:
Want to include a "Written by" tagline?

Reply:
I guess we don't have an About us page yet

Reply:
Still a good idea, I added one